docs/snapshots-restore.md

```diff
@@ -0,0 +1,13 @@
+# Restore snapshots
+
+We have created a docker image that contains the cardano-node together with a mithril-client. This image can be used to restore the latest snapshot for each network before starting the node. This image is automatically built and pushed to dockerhub every time there is an update to the relevant files on the repo provide several ways of using the snapshots for the supported components.
```

Comment:
I thought we were doing away with this image so we didn't have to continue maintaining it? Or am I misremembering something?

Either way, the instructions don't seem complete to me. It talks vaguely about the docker image but includes not specifics on how to actually use it or where to even find it.

Reply:
I think I've just moved some documentation I've found in the README.md and moved it here, I think we agreed to remove this image, I'm cleaning this up